youkube/test/YoukubeApplicationTestLegacy.kt

```diff
@@ -0,0 +1,139 @@
+import io.ktor.http.*
+import io.ktor.http.content.*
+import io.ktor.samples.youkube.*
+import io.ktor.server.config.*
+import io.ktor.server.testing.*
+import io.ktor.utils.io.streams.*
+import org.junit.Test
+import java.nio.file.*
+import kotlin.test.*
+
+/**
+ * Integration tests for the [main] module.
+ */
+class YoukubeApplicationTestLegacy {
+    /**
+     * Verifies that the [Index] page, returns content with "You need to upload some videos to watch them"
+     * for an empty test application.
+     */
+    @Test
+    fun testRootWithoutVideos() = testApp {
+        handleRequest(HttpMethod.Get, "/").apply {
+            assertTrue { response.content!!.contains("You need to upload some videos to watch them") }
+        }
+    }
+
+    /**
+     * Verifies the complete process of [Login] with the valid credentials (root:root) for this application,
+     * obtains the [Index] verifying that it now offers [Upload]ing files and that it links to the newly created [Video].
+     * Then it tries to access the [VideoPage] and ensures that it has a [kotlinx.html.VIDEO] element with the video.
+     *
+     * All this wrapped by a [cookiesSession] to be able to reuse [HttpHeaders.Cookie]/[HttpHeaders.SetCookie]
+     * among requests.
+     */
+    @Test
+    fun testUploadVideo() = testApp {
```

Comment:
is it possible to migrate this tests too?

Reply:
Do you mean the first `testRootWithoutVideos` test? Done.